tapeagents/observe.py

```diff
@@ -24,23 +24,17 @@
 TapeListener = Callable[[Tape], None]
 
 
-def init_sqlite_if_not_exists(only_once: bool = True):
+def migrate_sqlite_schema():
```

Comment:
We don't want to have the code to deal with obsolete schemas in the core frameworks. We have `tests/add_sqlite_columns.py` to patch the schemas offline. Please remove this.

Reply:
This is legacy code already removed. It looks like when I created my branch there was an issue with the number of columns in that table.